torch/fx/symbolic_trace.py

```diff
@@ -149,7 +149,7 @@ def module_call_wrapper(mod, *args, **kwargs):
                 return _create_proxy(self, 'call_module', module_qualified_name, args, kwargs)
         try:
             torch.nn.Module.__call__ = module_call_wrapper
-            self.graph.output(self.create_arg(fn(*args)))
+            self.output(self.create_arg(fn(*args)))
```

Comment:
This should call tracers create_node instead similar to how other things in symbolic trace work.